src/acl/external/time_quota/ext_time_quota_acl.8

```diff
@@ -45,6 +45,17 @@ then stderr will be used and the logging will go to Squids main cache.log.
 Enables debug logging in the logfile.
 .
 .if !'po4a'hide' .TP
+.if !'po4a'hide' .B "\-n"
```

Comment:
This is new feature with a relatively significant UI change. Please add it as a separate PR so it does not get lost.